test/lib/next-test-utils.ts

```diff
@@ -1036,8 +1023,6 @@ export async function getRedboxTotalErrorCount(
 }
 
 export async function getRedboxSource(browser: BrowserInterface) {
-  await ensureNoSuspendedComponentsInRedBox(browser)
```

Comment:
The contract is that you always call `assertHasRedbox` before which already waits for the suspended content.